### ISSUE TITLE
GLideN64 homepage preparations. Part I: compatibility list.

### ISSUE THREAD

**Issue by gonetz** [gonetz/GLideN64#2316]
I've got an proposition from EmuNations to host GLideN64 home page. It needs almost no efforts from my side, so I agreed.
Currently, I only need to "provide documentation on existing recommendations, tips & tricks in either excel, csv, google-doc format". GLideN64 never had a good compatibility list. Glide64 had some, so I took it as a base and put on google-doc:
https://docs.google.com/spreadsheets/d/1n3_wdI-iOGuLBro_9d96ZURau7cHElFwpv3yI-Juz_0/edit?usp=sharing

Everyone with the link can edit. If you are willing to help, please update the list. I was assured that the hosted list may contain screen shots and links to video files, so you may add them too.

**Comment by gonetz** [comment]
Unfortunately, nobody seems to be interested in preparing data for GLideN64 compatibility list.
Meanwhile, draft list is already on EmuNations:
https://www.emunations.com/gliden64-compatibility

Only "1080 Snowboarding" is more-less complete, as an example of how it may look.
Other games need updates. You may provide an update right on EmuNations - click on "Press here to update" button and fill the form.

**Comment by gonetz** [comment]
> I have recently started following PSX emulator Duckstation. It has a neat way of storing the compatibility list in xml format as part of the project sources. https://github.com/stenzek/duckstation/blob/master/data/database/compatibility.xml
> 
> I believe it could be nice to have a similar approach. Keep the xml file as part of the project and have the emunations website parse it. I added a simple [parser ](https://github.com/standard-two-simplex/GLideN64/tree/compatibility_list/compatibility-list)as proof of concept.

Emunations has a page for Duckstation with the compatibility list:
https://www.emunations.com/duckstation-compatibility

I'll ask if it uses Duckstation's compatibility list in xml format and can GLideN64 follow that scheme.


> Well, Emunations offers the possibility of adding/seeing images and videos after all.

Yes, and it is cool.

**Comment by gonetz** [comment]
> It has a neat way of storing the compatibility list in xml format as part of the project sources.

On the other hand, every update of the list will require new commit. It is not convenient.
The list on Emunations can be updated by community without involving me.
Probably it will be possible to fetch current state of the compatibility list from the Emunations in xml, html or any other format and put that list in CI builds.

**Comment by standard-two-simplex** [comment]
> On the other hand, every update of the list will require new commit. It is not convenient.

Yes, that is true. Perhaps it is best not to add anything until the list is more or less complete.

**Comment by oddMLan** [comment]
You could use the project's wiki, it doesn't require creating new commits for every edit.
And in the wiki you could link to Emunations for videos and pictures.
Emunations then can parse the wiki table format, which should be easy.

**Comment by Exequiel3k** [comment]
seems much more complicated than simply going to 1 place to have it all cleanly displayed. I don’t see the convenience of people being redirected to another website to view things.. It’s simpler to parse emunations which will have all the data consolidated than to ask emunations to parse a table while stitching images and videos of specific users submitting specific game updates while resolving and displaying all that within reasonable loading times..

**Comment by oddMLan** [comment]
> I don’t see the convenience of people being redirected to another website to view things..

I agree.

**Comment by weinerschnitzel** [comment]
I agree compatibility list should live in project wiki and parsed by website.

**Comment by Exequiel3k** [comment]
Both sides have their advantages. That said, we should just focus our energy on what this whole thread is about: updating the list in the GoogleDoc sheet..

**Comment by gonetz** [comment]
> Both sides have their advantages. That said, we should just focus our energy on what this whole thread is about: updating the list in the GoogleDoc sheet..

Yes, the main task is to prepare up-to-date data in a table form. The GoogleDoc sheet looks as a good variant. Its data can later be exported to Emunations, to GitHub wiki and so on.
